app/index.js

```diff
@@ -127,31 +128,6 @@ module.exports = generators.Base.extend({
       });
   },
 
-  removeFiles: function () {
-    var done = this.async();
-
-    var files = [
-      'package.json',
-      'bower.json',
-      'config/env/default.js'
-    ];
-
-    var remove = [];
-
-    for(var i = 0; i < files.length; i++) {
-      remove.push(exec('rm ./' + folder + '/' + files[i]));
-    };
-
-    Promise.all(remove)
```

Comment:
This still has to be fixed back to what it was. The new one has essentially duplicate code and would just take more time.